### PR TITLE
PYTHON-3399 Add launch.py to mongo-orchestration

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

File: mongo_orchestration/launch.py [hunk 1]
```diff
@@ -0,0 +1,354 @@
...
+            if data == 'q':
+                break
+            if data == 'r' and argv_has('repl'):
+                cluster.restart_primary()
+    finally:
+        cluster.stop()
+
+
+# Requires mongo-orchestration running on port 8889.
+# For SSL support you must modify the CERTS path.
```
Comment (ShaneHarvey) [inline mongo_orchestration/launch.py:334]: This line can be removed.

File: mongo_orchestration/launch.py [hunk 2]
```diff
@@ -0,0 +1,354 @@
...
+
+
+def main():
+    for arg in sys.argv[1:]:
+        try:
+            port = int(arg)
+            _free_port = itertools.count(port)
+        except:
+            pass
+    for version in ['2.6.12', '3.0.12', '3.2.10', '3.4.0-rc0']:
```
Comment (ShaneHarvey) [inline mongo_orchestration/launch.py:290]: -> `version in ['3.6', '4.0', '4.2', '4.4', '5.0', '6.0', '7.0', 'latest']:`

File: mongo_orchestration/launch.py [hunk 3]
```diff
@@ -0,0 +1,354 @@
+# Copyright 2015 MongoDB, Inc.
```
Comment (ShaneHarvey) [inline mongo_orchestration/launch.py:1]: 2015 -> 2023-present

**Review by ShaneHarvey — COMMENTED** [review]
Nice!

File: mongo_orchestration/launch.py (hunk 1 above)
Comment (blink1073) [reply mongo_orchestration/launch.py:334]: Done

File: mongo_orchestration/launch.py (hunk 2 above)
Comment (blink1073) [reply mongo_orchestration/launch.py:290]: Done

File: mongo_orchestration/launch.py (hunk 3 above)
Comment (blink1073) [reply mongo_orchestration/launch.py:1]: Done

**Review by ShaneHarvey — APPROVED** [review]

**Pull request merged**